offer-manager-domain/src/main/groovy/life/qbic/business/products/copy/CopyProduct.groovy

```diff
@@ -0,0 +1,12 @@
+package life.qbic.business.products.copy
+
+/**
+ * <class short description - One Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
+ * @since: <versiontag>
+ *
+ */
```

Comment:
```suggestion
/**
 * <h1>4.3.2 Copy Service Product</h1>
 * <br>
 * <p> As an offer admin, I want to create a new permutation of an existing product. 
 * <br> New permutations can include changes in unit price, sequencing technology and other attributes of service products.
 * </p>
 *
 * @since: 1.0.0
 */
```

Reply:
I changed the scentence a little and pushed it by hand :)

Reply:
8265081